### DISCUSSION TITLE
Deploying on-prem with no internet access

### DISCUSSION THREAD

**Discussion by ghost** [airbytehq/airbyte#45661] [Deployment]
Hi,

I was previously able to deploy via docker compose on one of our servers with no internet access by just scp'ing everything over that I needed.

Now that I'm trying to upgrade to use abctl, I'm having issues since it tries do pull the images it uses even if I've already loaded them into docker.

Is there a way I can use abctl on a server with no internet access?

**Comment by abuchanan-airbyte** [comment]
`abctl` uses Kind, which doesn't use images pulled to the host. We'd like to have better support built into abctl for what you're trying to do, but we're not there yet.

As a workaround, you can read the Kind docs about side-loading images into the container: https://kind.sigs.k8s.io/docs/user/quick-start/#loading-an-image-into-your-cluster

**Reply by ghost** [reply]
Thanks! I look forward to progress.

[a comment by an automated account is not shown]
